packages/angular_devkit/build_ng_packagr/src/build/index.ts

```diff
@@ -53,12 +88,30 @@ export class NgPackagrBuilder implements Builder<NgPackagrBuilderOptions> {
         ngPkgProject.withTsConfig(tsConfigPath);
       }
 
-      ngPkgProject.build()
-        .then(() => {
-          obs.next({ success: true });
-          obs.complete();
-        })
-        .catch((e) => obs.error(e));
+      if (options.watch) {
```

Comment:
Since ng-packagr `build()` uses `buildAsObservable()` internally why not remove the `if` and the `new Observable` statement and do `return ngPkgProject..withOptions({ watch: options.watch })buildAsObservable().pipe(map(() => {return { success: true };}))`

Reply:
The reason being is that angular cli still needs to support ng-package v2 and v3. And this method is only available in ng-packagr v4

Reply:
Never mind, I see, for backwards compatibility with ng-packagr 3.x when not using watch, right?

Reply:
Yes